hera_cal/delay_filter.py

```diff
@@ -27,8 +25,8 @@ class DelayFilter(VisClean):
     See vis_clean.VisClean for CLEAN functions.
     """
 
-    def run_filter(self, to_filter=None, weight_dict=None, horizon=1., standoff=0.15, min_dly=0.0, mode='clean',
-                   skip_wgt=0.1, tol=1e-9, verbose=False, cache_dir=None, read_cache=False, write_cache=False, **filter_kwargs):
+    def run_delay_filter(self, to_filter=None, weight_dict=None, horizon=1., standoff=0.15, min_dly=0.0, mode='clean',
```

Comment:
although this is technically a more informative name, i don't know if it is enough to warrant the API break. lmk your thoughts

Reply:
I don't think the function is exposed so I'm not too worried about breaking the API since I think the name is more specific and readable in the context of us now supporting both delay and fringe-rate domain high pass filtering. If this does break the pipeline then I do agree we should reconsider.

Reply:
OK. Here's an actual helpful response. 

Add a `run_filter` wrapper for `run_delay_filter` and a deprecation warning?

Reply:
Yeah I think that's a good idea, just have a shallow wrapper only in `DelayFilter.run_filter(**kwargs)` that then calls `self.run_delay_filter(**kwargs)` with a deprecation warning

Reply:
I've made this change.